app/docker/components/log-viewer/logViewer.html

```diff
@@ -15,6 +15,18 @@
               </label>
             </div>
           </div>
+          <div class="form-group">
+            <div class="col-sm-12">
+              <label for="tls" class="control-label text-left">
+                Display timestamps
+                <portainer-tooltip position="bottom" message="Display timestamps in log."></portainer-tooltip>
+              </label>
+              <label class="switch" style="margin-left: 20px;">
+               <input type="checkbox" ng-model="$ctrl.state.displayTimestamp" ng-change="$ctrl.logDisplayTimeStamp($ctrl.state.displayTimestamp)">
```

Comment:
Instead of relying on ng-change to update the value in service/container/task logsController, we're going to update the binded value directly.

Replace that input with the following:

```html
<input type="checkbox" ng-model="$ctrl.displayTimestamps"><i></i>
```